src/Servers/Kestrel/Transport.Abstractions/ref/Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions.netcoreapp3.0.cs

```diff
@@ -1,113 +1,3 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-namespace Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions.Internal
```

Comment:
I didn't realize you already deleted all of this already. How much more work is it to remove it entirely?

Reply:
I was going to remove it but it’s part of every solution because we have all of the dependencies there. I can do it less properly and just remove it from this solution (my guess is that nobody else did this yet?)